addons/mass_mailing/static/src/js/mass_mailing_snippets.js

```diff
@@ -274,6 +274,14 @@ options.userValueWidgetsRegistry['we-fontfamilypicker'] = SelectUserValueWidget.
 });
 
 options.registry.DesignTab = options.Class.extend({
+    /**
+     * @override
+     */
+    init() {
+        this._super(...arguments);
+        // Set the target on the whole editable so apply-to looks within it.
+        this.setTarget(this.$target.closest('body').find('.note-editable'));
```

Comment:
Why not `this.options.wysiwyg.getEditable()` like in your original code? Seems more elegant? And if not, finding the body with `this.$target.closest('body')` seems really strange, I think you could do `this.ownerDocument.body` or something like that

Reply:
You're absolutely right :facepalm: Done.